localization/english/names/wc_character_names_l_english.yml

```diff
@@ -18377,4 +18377,15 @@
  Eimear:0 "Eimear"
  Ugannar:0 "Ugann'ar"
  Eadric:0 "Eadric"
- 
\ No newline at end of file
+ Lilian:0 "Lilian"
+ Borugh:0 "Borugh"
+ Benedictus:0 "Benedictus"
+ Magroth:0 "Magroth"
+ Halahk:0 "Halahk"
+ Dagren:0 "Dagren"
+ Samuro:0 "Samuro"
+ Tagar:0 "Tagar"
+ Bovan:0 "Bovan"
+ Darkscale:0 "Lady Darkscale"
```

Comment:
Instead of localizing her full name as Lady Darkscale, maybe we should use flavorization to prefix her name with Lady and then make her first name Darkscale, or come up with a new name for her and make Darkscale her dynasty name.